modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -41,6 +41,44 @@ Generally, there is no need to download and install it separately, and we recomm
 There are binary Python wheels provided for Windows.
 
 
+== Version 3.2.5 (12 January 2022)
+
+[source,bash]
+----
+pip install couchbase==3.2.5
```

Comment:
The reason we do
python3 -m pip install couchbase==3.2.5
is for consistent success across platforms - RH in particular.